submissions/HannaSyn/memory-pair-game/app.js

```diff
@@ -0,0 +1,126 @@
+const characters = [
+  {
+    id: 1,
+    backImg: 'assets/Rick.png'
+  },
+  {
+    id: 2,
+    backImg: 'assets/Summer.png'
+  },
+  {
+    id: 3,
+    backImg: 'assets/Morty.png'
+  },
+  {
+    id: 4,
+    backImg: 'assets/missix.png'
+  },
+  {
+    id: 5,
+    backImg: 'assets/Jerry.png'
+  },
+  {
+    id: 6,
+    backImg: 'assets/Beth.png'
+  },
+  {
+    id: 7,
+    backImg: 'assets/pickle-rick.png'
+  },
+  {
+    id: 8,
+    backImg: 'assets/tele.png'
+  }
+]
+
+const playingField = document.querySelector('.playing-field');
+const cards = [...characters, ...characters];
+const cardsCollection = document.getElementsByClassName('card');
+const timeoutCheck = 1000;
+const timeoutReset = 2000;
+
+let firstClicked = null;
+let countCards = cards.length;
+
+const playingFieldBlocked = () => playingField.classList.add('blocked');
+const playingFieldUnblocked = () => playingField.classList.remove('blocked');
+const shuffleCards = () => cards.sort(function() { return 0.5 - Math.random() });
+
+function createCards() {
+  let card = '';
+  cards.forEach(({id, backImg}) => {
+    card += `<div data-id="${id}" class="card closed"><div class="front"><img src="assets/cover.jpg" alt="cover"></div><div class="back"><img src="${backImg}" alt="back"></div></div>`
+  })
+  playingField.innerHTML = card;
+};
+
+function fillField() {
+  shuffleCards();
+  createCards();
+}
+
+function resetField() {
+  playingField.innerHTML = '';
+  fillField();
+}
+
+function flipCards(e) {
+  let target = e.target;
```

Comment:
Try to use `function flipCards({target})`

Reply:
I'm sorry, but I don't understand, why it is better than `flipCards(e)` ?

Reply:
When drawing cards you used destructuring, here you can apply it too.

Reply:
Ok, thank you! Fixed